plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -413,6 +413,9 @@ func (p *pusher) convertEvent(e logs.LogEvent) *cloudwatchlogs.InputLogEvent {
 			// a valid timestamp and use the last valid timestamp for new entries that does
 			// not have a timestamp.
 			t = p.lastValidTime
+			if ((time.Now().UnixNano() / 1000000000) - (t / 1000)) > 432000 {
+				p.Log.Debugf("Unable to parse timestamp, using last valid timestamp %v: which is older than 5 days for log group %v: ", p.lastValidTime, p.Group)
```

Comment:
This seems like it should be a warning message. Also this will get logged for every log line we try to send. Could we throttle the amount of these?